Gemfile

```diff
@@ -9,7 +9,7 @@ gem 'nokogiri'
 ### Fastlane Plugins
 
 gem 'fastlane-plugin-sentry'
-gem 'fastlane-plugin-wpmreleasetoolkit', '~> 9.2'
+gem 'fastlane-plugin-wpmreleasetoolkit', '~> 11.0'
```

Comment:
See https://github.com/wordpress-mobile/release-toolkit/blob/trunk/MIGRATION.md

 - [x] Project uses ruby version `3.2.2` ➡️  [yes](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/.ruby-version)
 - [ ] `setbranchprotection` renamed to `set_branch_protection`, with `copy_branch_protection(from:…)` being usually even a better replacement for it ➡️ ❌  [Used in `fastlane/lanes/release.rb:79`](https://github.com/wordpress-mobile/WordPress-Android/blob/56f06862534d75f0374792115c51a10bbfbea8b8/fastlane/lanes/release.rb#L79)
 - [ ] `removebranchprotection` renamed `remove_branch_protection` ➡️ ❌  [Used in `fastlane/lanes/release.rb:296`](https://github.com/wordpress-mobile/WordPress-Android/blob/56f06862534d75f0374792115c51a10bbfbea8b8/fastlane/lanes/release.rb#L296)
 - [ ] `setfrozentag` renamed `set_milestone_frozen_marker`  ➡️ ❌  [Used in `fastlane/lanes/release.rb:80`](https://github.com/wordpress-mobile/WordPress-Android/blob/56f06862534d75f0374792115c51a10bbfbea8b8/fastlane/lanes/release.rb#L80) and [in `fastlane/lanes/release.rb:318`](https://github.com/wordpress-mobile/WordPress-Android/blob/56f06862534d75f0374792115c51a10bbfbea8b8/fastlane/lanes/release.rb#L318)
 - [x] ~`ios_clear_intermediate_tags` and `ios_final_tag` actions removed~: Not applicable for Android projects
 - [x] `has_alpha_version` removed ➡️ ✅ Not used
 - [x] `project_name` and `project_root_folder` removed ➡️ ✅ Not used
 - [x] `update_pull_requests_milestone` renamed to `update_assigned_milestone` (+ parameters renamed) ➡️ ✅ Not used
 - [x] ~`ios_check_beta_deps` parameter updated~: N/A to Android projects

Reply:
I've updated the calls to be in line with the `release-toolkit` version `11.0.2` (cc @wzieba)